### PR TITLE
Feature/hive-support

### PR DESCRIPTION
**Pull request by marktayl1**
Will merge after live notebook test with dogfood env if possible.

### CODE REVIEW

**Comment by wjohnson** [comment]
Oh man... this is so cool looking.
image

File: function-app/adb-to-purview/tests/tools/QualifiedNameConfigTester/config.json [hunk 1]
```diff
@@ -8,10 +8,28 @@
                 "op2": "test"
             }
         ],
-        "qualifiedName": "{prefix} : {nameSpcBodyParts[0]} : {nameSpcBodyParts[1]} : {nameSpcBodyParts[2]} : {nameSpcBodyParts[3]} : {nameSpcConParts[0]} : {nameSpcConParts[1]} : {nameSpcNameVals['one']} : {nameSpcNameVals['two']} : {nameGroups[0]} : {nameGroups[1]} : {nameGroups[0].parts[0]} : {nameGroups[0].parts[1]}",
+        "qualifiedName": " {prefix} : {nameSpcBodyParts[0]} : {nameSpcBodyParts[1]} : {nameSpcBodyParts[2]} : {nameSpcBodyParts[3]} : {nameSpcConParts[0]} : {nameSpcConParts[1]} : {nameSpcNameVals['one']} : {nameSpcNameVals['two']} : {nameGroups[0]} : {nameGroups[1]} : {nameGroups[0].parts[0]} : {nameGroups[0].parts[1]}",
```
Comment (wjohnson) [inline function-app/adb-to-purview/tests/tools/QualifiedNameConfigTester/config.json:11]: Do we need this space in front of the prefix?

**Review by wjohnson — COMMENTED** [review]
Functionally, I think everything is working. Just had a few questions on some edits to the docs pages.

File: docs/configuration.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ The following are the stable app settings that should be reviewed.
 |ClientID|Provided during ARM deployment|Yes|The Application Id of the service principal with permission to read call Databricks and Microsoft Purview APIs|
 |ClientSecret|Provided during ARM deployment|*Yes|The secret value for the service principal provided in ClientID|
 |Certificate|Not Applicable|*No|Should be a JSON object: `{"SourceType": "KeyVault","KeyVaultUrl": "https://akv-name.vault.azure.net/","KeyVaultCertificateName": "certificateName"}`
-|ResourceUri | https://purview.azure.com | No| |
-|AuthEndPoint| https://login.microsoftonline.com/| No| |
+|ResourceUri | <https://purview.azure.com> | No| |
```
[the remark this answers is not in the document]
Comment (marktayl1) [reply docs/configuration.md:14]: These prevent a markdown warning for no naked URLs

File: docs/configuration.md [hunk 3]
```diff
@@ -11,8 +11,8 @@ The following are the stable app settings that should be reviewed.
...
+|AuthEndPoint| <https://login.microsoftonline.com/>| No| |
```
[the remark this answers is not in the document]
Comment (marktayl1) [reply docs/configuration.md:15]: These prevent a markdown warning for no naked URLs

File: function-app/adb-to-purview/tests/tools/QualifiedNameConfigTester/config.json (hunk 1 above)
Comment (marktayl1) [reply function-app/adb-to-purview/tests/tools/QualifiedNameConfigTester/config.json:11]: No, that is just a typo.  I will change.

File: docs/configuration.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ The following app settings are experimental and may be removed in future release
 |Configuration|Value|Config Location| Note|
 |----|----|----|---|
 |spark.openlineage.url.param.code| Function Key |*Cluster Spark Config or Init Script||
-|spark.openlineage.host| https://FUNCTION_APP_NAME.azurewebsites.net|Cluster Spark Config or Init Script||
+|spark.openlineage.host| <https://FUNCTION_APP_NAME.azurewebsites.net>|Cluster Spark Config or Init Script||
```
[the remark this answers is not in the document]
Comment (marktayl1) [reply docs/configuration.md:48]: These prevent a markdown warning for no naked URLs

**Review by wjohnson — APPROVED** [review]
Great work, Mark!  This will be very helpful for many of customer's use caes.

**Pull request merged**